packages/analytics-js-integrations/__tests__/integrations/TikTokAds/tiktokads.test.js

```diff
@@ -186,6 +186,7 @@ describe('TiktokAds Track event', () => {
           content_type: 'CONTENT_TYPE',
           price: 50.0,
           quantity: 1,
+          brand: '',
```

Comment:
no data coming in brand, can you pls update the test input to get some valid data here?

Reply:
Done